tahoe_sites/zd_helpers.py

```diff
@@ -0,0 +1,16 @@
+"""
+Helpers used for Zero Downtime process. This file should be removed at the end
+"""
+from django.conf import settings
+
+
+def get_meta_managed():
+    return not settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS']
```

Comment:
Always set feature/settings default.

```suggestion
    return not should_site_use_org_models()
```